tests/wpt/mozilla/tests/mozilla/interfaces.worker.js

```diff
@@ -44,6 +44,7 @@ test_interfaces([
   "FocusEvent",
   "FormData",
   "HashChangeEvent",
+  "Headers",  
```

Comment:
This won't work yet, since the `Exposed=(Window,Worker)` annotation is commented out.